### ISSUE TITLE
URL gets "undefined" in link

### ISSUE THREAD

**Issue by phn2211** [strarsis/twentyseventeen-onepage#4]
hey strarsis, i'm happy that i found your onepage theme on github.
unfortunately i get the problem that "undefined" is automatically added to the url.
The link looks at the beginning like this "https://www.phn-consulting.de/site". After i hit a nav-link the template scrolls to the site, but it adds "undefined" to the URL. Following it looks like "https://www.phn-consulting.de/site/undefind#foto". I don't know where this addon comes from.

Can you check this please?

**Comment by strarsis** [comment]
@phn2211: I just checked the page with Chrome (on Windows 10) and it works fine.
Have you encountered this issue in a particular browser (Firefox, Internet Explorer, Edge, Opera)?

**Comment by phn2211** [comment]
Right, it's a microsoft edge problem. The pagerefresh (F5 Key) leads to an unknown site.

Firefox and chrome worke fine.

I have another 2 Questions:
1. Where can i setup max width mediaquery to swicht top navigion list to menu toggle. On tablet in vertical position the navbar is not visible. On mobile phone and desktop-pc it's available.

2. Where can i change the sequence of the pages?

Thanks four your help. This repo is amazing! :-)

**Comment by strarsis** [comment]
@phn2211:
I can confirm this issue in Edge. The `href` of the links is correct, scrolling to sections correctly changes the browser window URL. Clicking a link directly results in `undefined` being appended, so it has something to do with clicking on/selecting a link.

1. Toggling between mobile and tablet/desktop is handled by the 
[`@media screen and (min-width: 48em) { }`](https://github.com/WordPress/twentyseventeen/blob/master/style.css#L3224) (link) rule.
You would want to change it in the child theme, the easiest way would be copying the whole media query rule and pasting into your child theme, with `min-width` decreased to tablet widths.
Edit: Removed some incorrect parts.

2. You can change the sequence of the pages in Customizr Pages section.
There is no convenient drag and drop reorder implemented in twentyseventeen theme though,
but you can select the pages for new order from top to bottom 
(and leave the ones blank you don't need).
So you would select the first page, then the second page and so on 
from top to bottom thereby overwriting the old selection of pages.

**Comment by phn2211** [comment]
great tip :-) i tried and destroyed something. finally i installed a new wordpress to another folder to get a fresh style.css. it works like before now again. 

Sorry, i am really a beginner :-/ i can't handle your tip. in need to set up that max width 930px toggle-button and dropdown menu will be shown and min width 931px the whole nav-bar... 
how do i reach this. which classes do i have to "display:none" and "display:block". 

I am no developer or coder... just learning by doing and going on :-)

**Comment by strarsis** [comment]
@phn2211: The styles in this gist should be sufficient for keeping the full navigation menu 
(+ scroll down arrow): https://gist.github.com/strarsis/af0a5488ed08af5b0c8602202efdfb23
You would want to adjust the `min-width` for letting these styles apply earler (with less viewport width), 
so also tablets use the full navigation menu.

**Comment by phn2211** [comment]
thanks so much. i'll try tomorrow :-) site is not available at the moment.

**Comment by strarsis** [comment]
@phn2211: I reopened your issue because the `undefined` being appened to URL hadn't been resolved yet. 😄
